src/Maestro/tests/DependencyUpdateErrorProcessor.Tests/CreateGithubIssueTest.cs

```diff
@@ -0,0 +1,149 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Maestro.Data;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using Octokit;
+using Xunit;
+
+namespace DependencyUpdateErrorProcessor.Tests
+{
+    public class CreateGithubIssueTest : DependencyUpdateErrorProcessorTests
+    {
+        [Theory]
+        [InlineData("https://github.com/maestro-auth-test/maestro-test2", "38", "ProcessPendingUpdatesAsync", "no arguments" , "2200/1/1" , false)]
+        [InlineData("https://github.com/maestro-auth-test/maestro-test2", "38", "UpdateAssetsAsync", "[\"ee8cdcfb-ee51-4bf3-55d3-08d79538f94d\",\"UpdateAssetsAsync\"]",  "2200/1/1", false)]
+        public async Task ShouldCreateIssue(string repoUrl, string branch , string method, string arguments , DateTime errorOccurredAt, bool success)
+        {
+            Mock<RepositoryBranchUpdateHistoryEntry> repositoryBranchUpdate =
```

Comment:
Does this need to be a mock? We don't appear to be mocking anything interesting, just assigning a bunch of values to `Object`.  We should just be able to do
```csharp
var entry = new RepositoryBranchUpdateHistoryEntry {
    Repository = repoUrl,
    Branch = branch,
    //...
```

Reply:
There are actually a lot of mocks here that we only actually interact with the object inside's properties, so we could just use the object itself. Like there is no reason to mock any model types, or Octokit.Repsitory. They are just plain ol objects with settable properties.

Reply:
Sure will change this.